File: csharpmabye/TradersWrapper/Models/Faction.cs
public class FactionTrait
{
    public string? Name { get; set; }
    public string? Description { get; set; }
    public string? Symbol { get; set; }
}
public class Faction
{
    public string? Symbol { get; set; }
    public string? Name { get; set; }
    public string? Description { get; set; }
    public string? Headquarters { get; set; }
    public FactionTrait[]? Traits { get; set; }
    public bool IsRecuiting { get; set; }
}

File: backend/types/faction.go
package types

type FactionSymbols struct {
	Symbols []string `json:"symbols"`
}

type FactionTrait struct {
	Symbol      string `json:"symbol"`
	Name        string `json:"name"`
	Description string `json:"description"`
}

type Faction struct {
	Symbol       string         `json:"symbol"`
	Name         string         `json:"name"`
	Description  string         `json:"description"`
	Headquarters string         `json:"headquarters"`
	Traits       []FactionTrait `json:"traits"`
	IsRecruiting bool           `json:"isRecruiting"`
}
